release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -3576,4 +3576,23 @@ $ oc adm release info 4.8.48 --pullspecs
 
 ==== Updating
 
+To update an existing {product-title} 4.8 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster using the CLI] for instructions.
```

Comment:
I think that you can delete the "...for instruction" part of this sentence. It is implied.